meow/frontend/src/components/EditPost/EditForm.js

```diff
@@ -119,7 +120,20 @@ class EditForm extends React.Component {
     setFieldsValue({ tags });
   };
 
+  handleClickpub = e => {
+    this.setState({
+      ...this.state,
+      click_pubready: e.target.checked
+    });
+  };
+
   render() {
+    let flag_pub = null;
+    if (this.state.click_pubready && (this.props.story_url == "" || this.props.story_url == null)) {
+      flag_pub = <p className="no-url-error">Please make sure to enter a url</p>;
```

Comment:
Ahh sorry in the issue I think i forgot to update it and it should say that the url is optional but it should give a warning if there is no url. So I would recommend changing this this to be more of a warning than an error.